.changeset/fair-kangaroos-clean.md

```diff
@@ -0,0 +1,5 @@
+---
+"@blitzjs/codemod": patch
+---
+
+Fix for codemod that removes types from the blitz config file
```

Comment:
Remove TypeScript type annotations from `next.config.js` (a JavaScript file) in the `upgrade-legacy` codemod